src/components/SignIn/index.jsx

```diff
@@ -0,0 +1,128 @@
+import * as firebase from "firebase/app";
+import "firebase/auth";
+import "firebase/firestore";
+import React, { useState } from "react";
+import { useHistory } from "react-router-dom";
+import Form from "react-bootstrap/Form";
+import Button from "react-bootstrap/Button";
+import Row from "react-bootstrap/Row";
+import Alert from "react-bootstrap/Alert";
+import "./index.css";
+import { useTranslation } from "react-i18next";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faGoogle } from "@fortawesome/free-brands-svg-icons";
+import { faFacebook } from "@fortawesome/free-brands-svg-icons";
+
+const SignIn = () => {
+  const history = useHistory();
+  const [alert, setAlert] = useState(null);
+
+  var googleProvider = new firebase.auth.GoogleAuthProvider();
+  var facebookProvider = new firebase.auth.FacebookAuthProvider();
+
+  const handleFacebookLogin = (e) => {
+    firebase
+      .auth()
+      .signInWithPopup(facebookProvider)
+      .then(function (result) {
+        // var user = result.user;
+        setAlert({
+          message: "signIn.successMsg",
+          status: "success",
+        });
+        history.push("/");
+      })
+      .catch(function (error) {
+        setAlert({
+          message: error.message,
+          status: "danger",
+        });
+      });
+  };
+
+  const handleGoogleLogin = (e) => {
+    firebase
+      .auth()
+      .signInWithPopup(googleProvider)
+      .then(function (result) {
+        // var user = result.user;
+        setAlert({
+          message: "signIn.successMsg",
+          status: "success",
+        });
+        history.push("/");
+      })
+      .catch(function (error) {
+        setAlert({
+          message: error.message,
+          status: "danger",
+        });
+      });
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    firebase
+      .auth()
+      .signInWithEmailAndPassword(
+        e.target.elements[0].value,
```

Comment:
lets have these values saved inside a state, so the forms need to be controlled by these 2 states email and password